src/game_api/script/usertypes/state_lua.cpp

```diff
@@ -554,6 +555,15 @@ void register_usertypes(sol::state& lua)
     logicdiceshop_type["won_prizes_count"] = &LogicDiceShop::won_prizes_count;
     logicdiceshop_type["balance"] = &LogicDiceShop::balance;
 
+    /// Used in StateMemory
+    lua.new_usertype<RoomOwnersInfo>("RoomOwnersInfo", "owned_items", &RoomOwnersInfo::owned_items, "owned_rooms", &RoomOwnersInfo::owned_rooms);
+
+    /// Used in RoomOwnersInfo
+    lua.new_usertype<ItemOwnerDetails>("ItemOwnerDetails", "owner_type", &ItemOwnerDetails::owner_type, "owner_uid", &ItemOwnerDetails::owner_uid);
+
+    /// Used in RoomOwnersInfo
+    lua.new_usertype<RoomwOwnerDetails>("RoomwOwnerDetails", "layer", &RoomwOwnerDetails::layer, "room_index", &RoomwOwnerDetails::room_index, "owner_uid", &RoomwOwnerDetails::owner_uid);
```

Comment:
Oh looks like Roomw has a typo everywhere and I didn't notice a single one when I checked this earlier